vm/2.0/modules/ROOT/pages/vm-studio-configuration.adoc

```diff
@@ -0,0 +1,137 @@
+. <<configure-global-element,Define the VQ queue>>
+. On the *Listener* configuration screen, set the *Number of consumers* field to your desired value. Defaults to `4`. +
+. In the *Queue* section, select a *Queue name* from the drop-down box. +
+If you previously created a VM queue, the *Queue name* value appears automatically.
+. Set the *Timeout* to your desired value. Defaults to `5`.
+. Select a *Timeout unit* from the drop-down box.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for VM Connector, follow these steps:
+
+. In the *Mule Palette* view, select *VM Connector* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. A VM Connector global configuration defines the queues on which the connector operates. You can define any number of configurations, each with its own set of queues. +
+
+Each queue defined within a given VM Connector configuration can be used only by operations referencing that specific configuration. A *Queue name* cannot be repeated across configurations, and two queues with the same name can not exist in the same app or domain.
+
+To configure the global element for VM Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. In the *General* section, for the *Queues* drop-down box select either:
+
+* `Bean reference` +
+.. In the *Bean reference* field, add your desired reference value.
+* `Edit inline` +
+.. In the *Queues* box, click the plus sign (+) to add a new queue.
+.. In the *Queue* window, add a *Queue name*, set *Queue type* to either `TRANSIENT (Default)` or `PERSISTENT`, and add a value for *Max outstanding messages*.
+.. Click *Finish*.
+
+[start=4]
+. In the *Connection* section, optionally configure a reconnection strategy.
+. On the *Advanced* tab, optionally configure an expiration policy.
+. Click *OK*.
+
+The following screenshot shows an example of configuring the VM Connector:
+
+.VM Connector Configuration
+image::vm-studio-config1.png[VM Connector Configuration]
+
+The following screenshot shows an example of configuring the VM queue:
+
+.VM Queue Configuration
+image::vm-studio-config2.png[VM Queue Configuration]
+
+In the XML editor, the VM Connector configuration looks like this:
```

Comment:
"the" is OK here because it refers to "configuration", not "VM Connector" (per edit Line 31)